src/components/Articles.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { getArticles } from "../api";
+import PlaceholderImage from "../misc/elementor-placeholder-image.jpg";
+
+export default function Articles() {
+    const [articles, setArticles] = useState([]);
+    useEffect(() => {
+        getArticles().then((articles) => {
+            setArticles(articles);
+        });
+    }, []);
+    return (
+        <ul className="Articles">
+            {articles.map((article) => {
+                return (
+                    <li key={article.article_id}>
+                        <Link to={`/articles/${article.article_id}`}>
+                            <img src={PlaceholderImage} alt="placeholder" />
```

Comment:
the article objects will have urls for images on them, so let's use them here.

also need something more descriptive for the image alts